app/models/import/builder.rb

```diff
@@ -0,0 +1,13 @@
+class Import
+  class Builder
+    def initialize(gqueries, id:, scaling:, **)
+      @gqueries    = gqueries
+      @scenario_id = id
+      @scaling     = scaling
+    end
+
+    def build(response = nil)
```

Comment:
Aha, that makes sense. I was already messing around with what to put where. However, I sort of do disagree with just putting an underscore to the `_response` in for instance the `HouseBuilder`. The specs will than look like: 

````ruby
Import::HouseBuilder.new(gqueries, { ... }).build(nil) 
````
Which looks a bit funky in my opinion. But 

````ruby
def build(_response = nil)
   ....
end
````

Also looks really odd.